darts/tests/models/forecasting/test_regression_ensemble_model.py

```diff
@@ -173,15 +224,154 @@ def test_torch_models_retrain(self):
         )
 
         ensemble = RegressionEnsembleModel([model1], 5)
+        # forecasting model is retrained from scratch with the entire series once the regression model is trained
         ensemble.fit(self.combined)
+        model1_fitted = ensemble.forecasting_models[0]
+        forecast1 = model1_fitted.predict(3)
+        # train torch model outside of ensemble model
+        model2.fit(self.combined)
+        forecast2 = model2.predict(3)
 
-        model1_fitted = ensemble.models[0]
-        forecast1 = model1_fitted.predict(10)
+        assert model1_fitted.training_series.time_index.equals(
+            model2.training_series.time_index
+        )
+        assert forecast1.time_index.equals(forecast2.time_index)
+        np.testing.assert_array_almost_equal(forecast1.values(), forecast2.values())
 
-        model2.fit(self.combined)
-        forecast2 = model2.predict(10)
+    @pytest.mark.parametrize("config", [(1, 1), (5, 2), (4, 3)])
+    def test_train_with_historical_forecasts_no_covs(self, config):
+        """
+        Training regression model of ensemble with output from historical forecasts instead of predict should
+        yield better results when the forecasting models are global and regression_train_n_points >> ocl.
+
+        If the ocl of the forecasting models is not a multiple of the regression_train_n_points, the regression
+        model might be trained with less points.
```

Comment:
This paragraph should be removed since the first points of the direct prediction as now used as "padding"